### ISSUE TITLE
Adjust layout of /news page for improved featured image sizes.

### ISSUE THREAD

**Issue by anniehirshman** [GetOrganizedBrooklyn/getorganizedbk-website#22]
Adjust layout of https://getorganizedbk.org/news/ page so the featured image does not take up the entire page (desktop viewport widths).  

Problem: 
Currently, on desktop, the featured images of blog posts take up the entire screen - so the user doesn't see any text when they arrive on the News page, and have to scroll a lot to see multiple posts. 

Proposed Solutions: 
1) Ideally, adjust design of page and create a 2- or 3-column layout, which would reduce featured image sizes and display more posts at larger screen sizes.  
2) If creating a new layout isn't possible, then reduce max width of News feed content area.

Note: smaller screen sizes (mobile/tablet) are currently working well and should not be affected if possible.

**Comment by JpTiger** [comment]
Technically fixed via solution 1, changing the layout to 2-column. Does leave an awful lot of whitespace. I think an archive widget or sidebar of some kind to the right side wouldn't look half bad, but I'm not seeing something premade for the purpose in Divi.

**Comment by JpTiger** [comment]
...so I made one myself! I created a new Widget area for "News Sidebar" and tossed a couple things in. It does behave a bit weirdly, I started by putting in both a news archive and the events, but they get displayed side-by-side instead of one above the other, and they use different fonts by default so the whole thing looked a bit odd. So I cut out the events part. 

If you want to edit the sidebar, you'll want to do it both on the page and then in the Appearance > Widget section of WordPress. Look for the "News Sidebar" area at the bottom.
